proto/provider/akash/manifest/v2beta3/service.proto

```
@@ -37,6 +37,23 @@ message ServicePermissions {
+// TEEParams configures Trusted Execution Environment for the service.
+// The type field selects the TEE technology and maps to a Kata runtime class.
+// The attestation field controls whether the provider injects an attestation sidecar.
+message TEEParams {
+  // type is the TEE technology: sev-snp, sev-snp-gpu, tdx, tdx-gpu
+  string type = 1 [
+    (gogoproto.jsontag)  = "type",
+    (gogoproto.moretags) = "yaml:\"type\""
+  ];
+  // attestation controls whether the provider injects an attestation sidecar.
+  // Defaults to true. Set to false to bring your own attestation.
+  bool attestation = 2 [
```

Buf (1.69.0)

[error] cannot find `gogoproto.nullable` in this scope

(COMPILE)